src/TodoList.js

```diff
@@ -1,25 +1,12 @@
 import React from "react";
 import TodoListItem from "./TodoListItem";
 
-let todoList = [
-  {
-    id: 1,
-    title: "complete assignment",
-  },
-  {
-    id: 2,
-    title: "Cook dinner",
-  },
-  {
-    id: 3,
-    title: "Do laundry",
-  },
-];
-
-function TodoList() {
+function TodoList({ todoList }) {
+  let todos = todoList;
```

Comment:
Is there a reason to reassign this?  You should just use the `todoList` variable.

Sidenote, were you to leave it like this, since you're not reassigning it, you should have used `const` instead of `let`

Reply:
Ok thanks. I get it